sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/_version.py

```diff
@@ -0,0 +1,2 @@
+# matches SEMVER
+VERSION = "0.0.18b3"
```

Comment:
This specific version number seems to be a default, but it should be replaced with the same version as in the changelog. In this case, it would be:
```suggestion
VERSION = "1.0.0b1"
```